### ISSUE TITLE
Document how to setup and run database extension tests

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#6414]
There's currently no developer documentation on which database tests are enabled/disabled and how to set up and run the database tests which are disabled by default. We should add at least few sentences on on this topic to the developer documentation.

**Comment by Abhishek-dev1** [comment]
Hi @tfmorris   ,Could you assign this task to me i would like to work on this issue? Thanks

**Comment by wetneb** [comment]
I would intuitively document this in the technical documentation hosted on openrefine.org, so I would transfer this issue to https://github.com/OpenRefine/openrefine.org, unless you'd expect it to be documented somewhere else @tfmorris?

**Comment by tfmorris** [comment]
> I would intuitively document this in the technical documentation hosted on openrefine.org, so I would transfer this issue to https://github.com/OpenRefine/openrefine.org, unless you'd expect it to be documented somewhere else @tfmorris?

Yes, that's fine. I just forgot that the developer documentation is a different place than the code. Perhaps you should get rid of the `current docs` and `Type: Documentation` labels too.
